### ISSUE TITLE
questions about G's loss

### ISSUE THREAD

**Issue by woshichunge12** [odegeasslbc/FastGAN-pytorch#36]
According to the formula of your loss function, the loss of G is the opposite of the score given by D. Since D will only give a positive number, why does the loss of G need to be added with a negative sign?
![image](https://user-images.githubusercontent.com/40681348/141642044-ca7d07ce-d485-4270-9de4-7c6636c1430f.png)
What makes me even more strange is that when the G loss is printed in the code, a negative sign is added, but when I run it, the output is a negative number, indicating that the loss of G should be a positive number, but according to the formula, it cannot be a positive number.
`        if iteration % 100 == 0:
            print("GAN: loss d: %.5f    loss g: %.5f   " %(err_dr, -err_g.item()))`
![image](https://user-images.githubusercontent.com/40681348/141642151-b6763b64-60c5-4db4-89c4-a2350f155ed5.png)

**Comment by woshichunge12** [comment]
@odegeasslbc please help  thanks very much~

**Comment by odegeasslbc** [comment]
Hi, the formula is correct, and the code to print out the loss is wrong (the neg sign should not be added, but it won't affect the actual training anyway).

The reason for the neg sign in the formula: we want to maximize E| D(G(x)) |, to we minimize - E| D(G(x)) | .
